Dockerfile

```diff
@@ -1,3 +1,9 @@
 FROM nginx
 LABEL name="verapdf-webapp-gui"
-COPY ./landing /usr/share/nginx/html
\ No newline at end of file
+# Remove default nginx configuration
+RUN rm /etc/nginx/conf.d/default.conf
```

Comment:
Let's probably not remove default config file. Instead let's define our site config in a separate file and put it into `sites-enabled/` folder which should be picked up by default config.